rustfmt-config/src/options.rs

```diff
@@ -189,6 +189,15 @@ configuration_option_enum! { Color:
     Auto,
 }
 
+configuration_option_enum! { LineWidth:
+    // Always format lines as short as possible (?)
+    Minimum,
```

Comment:
The `null` width heuristic doesn't make lines as short as possible, it tries to layout stuff in its 'default' layout without taking into consideration the width of the sub-item. I.e., it will make function calls as wide as possible, but will never put if/else on one line (since multi-line is the default).